src/caffegraphinput.cc

```diff
@@ -181,15 +183,15 @@ namespace dd
   {
     caffe::NetParameter net;
     if (!read_proto(filename, &net))
-      return -1;
+      throw MLLibBadParamException("unable to parse protofile");
```

Comment:
In my understanding, this function both returns an error code *and* throws exceptions? Do we need to keep the error code, now it's always 0?

Reply:
ok, removed error code